packages/mobile/src/localization/languages/en/common.ts

```diff
@@ -10,4 +10,20 @@ export const common = {
   common: {
     HOME: 'Home',
   },
+  services: {
+    ERROR: 'Error',
```

Comment:
I more relates to common workspace, it has nothing specific

Reply:
fixed